app/scripts/lib/auth-errors.js

```diff
@@ -339,6 +339,14 @@ define(function (require, exports, module) {
       errno: 1048,
       // not user facing, only used for logging, not wrapped in `t`
       message: 'Cannot change input type: %(type)s'
+    },
+    PHONE_NUMBER_REQUIRED: {
```

Comment:
This error is thrown by the [tel-input](https://github.com/mozilla/fxa-content-server/pull/4625/files#diff-77f062a25a042bccef940a2b7ebdc313) which in theory could be used anywhere.